cmake/fuzzing-murxla.cmake

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+# Top contributors (to current version):
+#   Gereon Kremer
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# Find Murxla
+# Murxla_FOUND - system has Murxla
+##
+
+if(NOT IS_DIRECTORY "${CMAKE_BINARY_DIR}/murxla")
+  # this is only necessary while the murxla repository is private
+  add_custom_target(fuzz-murxla
+    COMMAND echo "To enable make fuzz-murxla run"
+    COMMAND echo "git clone git@github.com:murxla/murxla.git"
+  )
+  return()
+endif()
+
+include(ExternalProject)
+
+set(Murxla_COMMIT "afc5744766d6aa61ad5b7ea27007666ac7a5aec2")
+
+add_custom_target(install-for-murxla
+  COMMAND ${CMAKE_MAKE_PROGRAM} install DESTDIR=murxla-install
+  DEPENDS cvc5-bin
+)
+
+ExternalProject_Add(
+  Murxla-EP
+  EXCLUDE_FROM_ALL ON
+  ${COMMON_EP_CONFIG}
+  #URL https://github.com/murxla/murxla/archive/${Murxla_COMMIT}.tar.gz
+  #URL_HASH SHA1=da39a3ee5e6b4b0d3255bfef95601890afd80709
+  SOURCE_DIR ${CMAKE_BINARY_DIR}/murxla
+  CMAKE_ARGS
+    -DCMAKE_PREFIX_PATH=${CMAKE_BINARY_DIR}/murxla-install/usr/local/lib/cmake/cvc5/
```

Comment:
Hmm that's weird, when I configure murxla with I point `CMAKE_PREFIX_PATH` to the install directory that contains `bin`, `include`, `lib` and it works.

Reply:
Note that the directory does *not* contain bin, include and lib, all this stuff is below `usrc/local/`. This is because I call `${CMAKE_MAKE_PROGRAM} install DESTDIR=murxla-install` (instead of tainting the cmake config with setting the prefix dir). It may work with `usr/local` instead of the full `usr/local/lib/cmake/cvc5`...

Reply:
Indeed `usr/local` is sufficient.